application/deed/utils.py

```diff
@@ -0,0 +1,32 @@
+import os
+from jsonschema.validators import validator_for
+import json
+
+
+def validate_helper(json_to_validate):
+    error_message = ""
+    error_list = sorted(_title_validator.iter_errors(json_to_validate),
+                        key=str, reverse=True)
+
+    for count, error in enumerate(error_list, start=1):
+        error_message += "Problem %s:\n\n%s\n\n" % (count, str(error))
+
+    return len(error_list), error_message
+
+
+def _load_json_schema():
+    script_path = os.path.dirname(os.path.realpath(__file__))
```

Comment:
I think there are pro's and con's to relative and absolute paths. The decision to which is preferable depends upon whether the resource being targeted is organised globally e.g. you plan to have a /schema directory somewhere - the benefit being the source code can be relocated and the targeted file location will still be valid. The other situation where the resource is local and if the outer container/folder is moved with the resource then the path is still valid. 
I think both ways are equally readable, the issue is how the target resource is organised to best survive project refactors.